albums.js

```diff
@@ -989,8 +989,23 @@ var albumData = {
 			"album_title": "Synchronize (VIP Mix / Acoustic) EP",
 			"album_artwork_thumb": "https://i.imgur.com/S1uMtsMm.jpg",
 			"album_artwork_full": "https://i.imgur.com/S1uMtsM.jpg"
+		},
+
+		{
+			"album_number": "MCEP098",
```

Comment:
The albums here need to be listed in chronological order in order for it to appear correctly on the site.